src/components/UiCardList/cardData.tsx

```
@@ -0,0 +1,59 @@
+import { Card } from '@/features/landing/types/Card/card-item';
```

ESLint

[error] '@/features/landing/types/Card/card-item' import is restricted from being used by a pattern.

(no-restricted-imports)